shared/src/sector/registered_proof.rs

```diff
@@ -33,23 +33,13 @@ pub enum RegisteredSealProof {
 
 impl RegisteredSealProof {
     /// Returns registered seal proof for given sector size
-    pub fn from_sector_size(size: SectorSize, network_version: NetworkVersion) -> Self {
-        if network_version < NetworkVersion::V7 {
-            match size {
-                SectorSize::_2KiB => Self::StackedDRG2KiBV1,
-                SectorSize::_8MiB => Self::StackedDRG8MiBV1,
-                SectorSize::_512MiB => Self::StackedDRG512MiBV1,
-                SectorSize::_32GiB => Self::StackedDRG32GiBV1,
-                SectorSize::_64GiB => Self::StackedDRG64GiBV1,
-            }
```

Comment:
ref-fvm cannot service nv7, and nv7 actors were Go specs-actors, so this can be removed.